src/main/java/org/carlmontrobotics/Constants.java

```diff
@@ -4,56 +4,235 @@
 
 package org.carlmontrobotics;
 
-import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import org.carlmontrobotics.lib199.Limelight.Transform;
+import org.carlmontrobotics.lib199.swerve.SwerveConfig;
+
+
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.RobotBase;
+import edu.wpi.first.wpilibj.XboxController.Axis;
+import edu.wpi.first.wpilibj.XboxController.Button;
 
 /**
- * The Constants class provides a convenient place for teams to hold robot-wide numerical or boolean
- * constants. This class should not be used for any other purpose. All constants should be declared
+ * The Constants class provides a convenient place for teams to hold robot-wide
+ * numerical or boolean
+ * constants. This class should not be used for any other purpose. All constants
+ * should be declared
  * globally (i.e. public static). Do not put anything functional in this class.
  *
- * <p>It is advised to statically import this class (or one of its inner classes) wherever the
+ * <p>
+ * It is advised to statically import this class (or one of its inner classes)
+ * wherever the
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+	public static final double g = 9.81; //meters per second squared
+
 	public static final class Drivetrain {
+
+			//#region Subsystem Constants
+
+			public static final double wheelBase = Units.inchesToMeters(16.75);
+			public static final double trackWidth = Units.inchesToMeters(23.75);
```

Comment:
Delete Drivetrain constants if you are not using them. Dealing with merge conflicts when we eventually have to merge code together is annoying